solution.rb

```diff
@@ -0,0 +1,8 @@
+require_relative "lib/diff"
+
+file1 = Diff::FileSource.read("./file1.txt")
+file2 = Diff::FileSource.read("./file2.txt")
+
+diff =  file1.diff_with(file2)
+
+File.write("solution.txt", diff.join("\n"))
```

Comment:
I do not feel that the output method belongs in the core codebase which is why it is here.